src/package.rs

```diff
@@ -200,6 +204,20 @@ fn exec_cargo(
     line.exec()
 }
 
+fn cargo_clean(cargo: &OsStr, args: &Args, package: &Package<'_>) -> Result<()> {
+    let mut line = ProcessBuilder::new(cargo, args.verbose);
+    line.arg("clean");
+    line.arg("--package");
+    line.arg(&package.name);
+
+    if args.verbose {
+        // running `cargo clean --package <package>`
+        info!(args.color, "running {}", line);
+    }
```

Comment:
It doesn't display by default, but it's probably fine. (displayed when you pass the `-v` flag.)